PoGo.Necrobot.Window/Model/GymViewModel.cs

```diff
@@ -30,19 +30,41 @@ public string GymIcon
             {
                 string fortIcon = "";
                 bool isRaid = false;
+                bool asBoss = false;
+
                 try
                 {
-                    if (!string.IsNullOrEmpty(fort.RaidInfo.RaidPokemon.PokemonId.ToString()))
+                    if (fort.RaidInfo.RaidBattleMs > 0)
                         isRaid = true;
                 }
                 catch
                 {
                     isRaid = false;
                 }
-                string gymStat = isRaid ? "-raid" : "";
+
+                try
+                {
+                    if (!string.IsNullOrEmpty(fort.RaidInfo.RaidPokemon.PokemonId.ToString()))
+                        asBoss = true;
+                }
+                catch
+                {
+                    asBoss = false;
+                }
+
+                string gymStat = isRaid ? "-raid" : null;
+                string gymBoss = asBoss ? $"https://cdn.rawgit.com/Necrobot-Private/PokemonGO-Assets/master/pokemon/{(int)fort.RaidInfo.RaidPokemon.PokemonId}.png" : null;
+
+                // Solution to overlay 2 images in string mode ????
+                // forticon + gymBoss or create asset 251 gyms red + 251 gyms blue + 251 neutral +251 yellow !!!!
+                // WFP is compatible to var image ??? 
+
```

Comment:
Ok, rocket uses a procedure or "function" in /rocketbot2/helpers/resourcehelper.cs to combine 2 images This is used for "slashed" pokemons and for gym image creation with raidpokemon (boss), see here: https: //github.com/Necrobot-Private/NecroBot/blob/master/RocketBot2/Helpers/ResourceHelper.cs

These functions give as an image a return and not a string

"Public Image CombineImages (Image image, Image Image2)"
Image = equal to gym image
Image2 = gym boss image
These images are superimposed one on top of the other and first the gym after the pokemon so that the gym image does not hide the image pokemon (boss) and this returns Image.

The same goes for the "slashed" pokemon

My problem is that WPF uses string instead of Image in return, I wonder if there is any way to combine 2 strings of the typo link as WPF uses in one image.

As you can see below in the image the Gym with the Boss inside
And the Pokémon slashed:
![sans_titre](https://user-images.githubusercontent.com/11375035/28069954-9a7dd6c6-664b-11e7-8e1a-3ba5f4273618.png)

A necro gui wpf how does not combine images the slashed it uses is
![image](https://user-images.githubusercontent.com/11375035/28070073-0280ba68-664c-11e7-9cf5-3becd662161b.png)